packages/eas-cli/src/graphql/queries/UpdateQuery.ts

```diff
@@ -8,7 +8,10 @@ import {
   ViewBranchUpdatesQueryVariables,
 } from '../generated';
 
-const PAGE_LIMIT = 300;
+export const PAGE_LIMIT = 300;
```

Comment:
Give this constant a more global, less context-sensitive name since it's now exported (it's not a global page limit we use everywhere). Also, the name should indicate it's a limit on the number of updates per page (vs, say, a limit on the number of pages).